### DISCUSSION TITLE
Failed to acquire installed data lock, only one instance of Legendary may install/import/move applications at a time

### DISCUSSION THREAD

**Discussion by trixyder** [RareDevs/Rare#329] [General]
i want to download a game and suddenly have this. What shoould i do?
![Снимок](https://github.com/RareDevs/Rare/assets/153106534/833a71e9-bf8f-431a-b86b-5bcc1a806637)

**Comment by loathingKernel** [comment]
Are you doing anything else at the same time through Rare? For example verifying, importing or installing something?
